pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -66,7 +66,7 @@
 
     <!-- simple check to make sure we don't accidentally pull an old 
          package that should have been merged to the primary package -->
-    <Error Text="Unexpected System package %(_secondaryFilesByPackagePath.NuGetPackageId)" Condition="$([System.String]::new('%(_secondaryFilesByPackagePath.NuGetPackageId)').Contains('System.'))" />
+    <Error Text="Unexpected System package %(_secondaryFilesByPackagePath.NuGetPackageId)" Condition="$([System.String]::new('%(_secondaryFilesByPackagePath.NuGetPackageId)').Contains('System.')) and ('%(_secondaryFilesByPackagePath.NuGetPackageId)' != 'System.ValueTuple')" />
```

Comment:
I was running into some problems, Eric with the above.  I think the problem may be due to the fact that we want to use the NuGetPackageId to do the filtering instead of Identity (I was finding cases where runtime packages which contributed resources named "System" were getting flagged).  I'll take a look with you in person tomorrow.  Simply changing the metadata we do the Condition on didn't causes the task to fail when it *should have* so I'll need some more MSBuild help, I fear.